services/QuillLMS/app/services/classlink_integration/auth_credential_saver.rb

```diff
@@ -27,8 +33,12 @@ def fetch_credentials
       Net::HTTP.post_form(uri, form_data)
```

Comment:
Also, is there a reason we use `Net::HTTP` here and `HTTParty` in other places seems like we should use one (I believe we use `HTTParty` in most of our new code.)